packages/presentation/lib/style/colors.dart

```diff
@@ -1,5 +1,4 @@
 class MyColors {
   static int get splashScreenUp => 0xffE51937;
-
   static int get splashScreenBottom => 0xffDB5252;
```

Comment:
-//-

Reply:
here it's up to u. I also prefer const in this case